keyboards/mechlovin/olly/jf/keymaps/via/config.h

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2021 mechlovin
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define DYNAMIC_KEYMAP_EEPROM_MAX_ADDR 1607
+#define DYNAMIC_KEYMAP_LAYER_COUNT 6
```

Comment:
```suggestion
```

Reply:
The Olly JF is made compatible with the Duck Jetfire, which has 5 layer-indicating LEDs, so I think 6 layers in VIA is suitable. Please let me know if that's ok. Thank you.

Reply:
Then the keymap needs to match.

Reply:
Added. Thank you.